src/overlays/actors/ovl_En_Kusa/z_en_kusa.c

```diff
@@ -8,38 +8,43 @@
 #include "overlays/effects/ovl_Effect_Ss_Kakera/z_eff_ss_kakera.h"
 #include "objects/gameplay_keep/gameplay_keep.h"
 #include "objects/gameplay_field_keep/gameplay_field_keep.h"
+#include "objects/object_kusa/object_kusa.h"
 #include "vt.h"
 
 #define FLAGS 0x00800010
 
 #define THIS ((EnKusa*)thisx)
 
+#define TYPE(params) ((params) & 3)
+#define SPAWN_BUGS ((this->actor.params >> 4) & 1)
+#define DROPS ((this->actor.params >> 8) & 0xF)
```

Comment:
```suggestion
#define ENKUSA_TYPE(params) ((params) & 3)
#define ENKUSA_SPAWN_BUGS(params) ((params >> 4) & 1)
#define ENKUSA_DROPS(params) ((params >> 8) & 0xF)
```
We haven't technically agreed to a style for `params` macros, but I feel like writing them like this is more appropriate for now.

Reply:
agreed. would also prefer we just wait till we standardize a way to do this, as mentioned in the other pr